draft-ietf-rats-corim.md

```diff
@@ -1300,6 +1300,35 @@ they are not required to use the same internal data structures.
 For example, it is expected that the resources used during the initialisation
 phase can be amortised across multiple appraisals.
 
+## Verifier Abstraction
+
+This document assumes Verifier implementations may differ.
+To facilitate description of normative Verifier behavior,
+this document uses abstract representation of Verifier internals.
+
+* Claim: A structure that has a key-value pairing.
+* Environment Measurement Tuple (EMT): A structure containing a set of environment
+Claims that describe a Target Environment and a set of measurement Claims that
+describe attributes of the Target Environment.
+* possible state: Claims that describe various alternative forms of a Target Environment.
```

Comment:
Maybe should inherit `intended state` here? (there can be various allowed intended states)

Reply:
See below: following Dave's wording for now.